src/TwoSlotDisplay.js

```diff
@@ -34,11 +42,13 @@ class TwoSlotDisplay extends Component {
 
   render() {
     let next = getNextSlot(this.state.day, this.state.slot);
+    const weekend = this.state.day === 0 || this.state.day === 6;
```

Comment:
This check shouldn't happen here again. You should instead pass in the `isWeekend` boolean in the React properties. The `render` function shouldn't have to worry about what a weekend is.